Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -432,6 +447,33 @@ public extension Persistence {
                 }
             }
         }
+
+        private func makeRemoveAllOperation(completion: @escaping WriteCompletionClosure) -> DiskMemoryBlockOperation {
+            return DiskMemoryBlockOperation() { [unowned self] in
+
+                let configuration = self.configuration
+
+                do {
+                    try configuration.fileManager.removeItem(atPath: configuration.path)
+                } catch {
+                    completion(.failure(.failedToRemoveAll(.removeDirectoryFailed(error))))
+                    return
+                }
+
+                self.usedDiskSize.value = 0
+
+                do {
+                    try configuration.fileManager.createDirectory(atPath: configuration.path,
```

Comment:
Maybe use `createDiskCacheDirectory()` ?

Reply:
I purposely didn't so that the returned error would be different (i.e. would be nested in `Error.failedToRemoveAll`)